packages/simple-types/test/notebook.test.ts

```diff
@@ -1,14 +1,14 @@
 /** @license
  * Copyright 2020 Esri
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Version 2.0 (the 'License');
  * you may not use this file except in compliance with the License.
```

Comment:
Just to know, the original statement is the format that was specified by Ben Parker and Keith Ryden back in 2012; it wasn't a style choice on our part.

Reply:
Thanks for the info - it's always interesting on this stuff - hub.js does not have any licensing info in the individual files, and rest-js just has this header

```
/* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
 * Apache-2.0 */
```

The UMD builds for all of them inject a different license header, the ESM and CJS outputs keep the file headers if they exist... maybe we need to connect with Keith and Ben re: what should be in Hub.js :)